src/main/java/com/algorand/algosdk/builder/transaction/AssetAcceptTransactionBuilder.java

```diff
@@ -0,0 +1,97 @@
+package com.algorand.algosdk.builder.transaction;
+
+import com.algorand.algosdk.crypto.Address;
+import com.algorand.algosdk.transaction.Transaction;
+
+import java.math.BigInteger;
+import java.security.NoSuchAlgorithmException;
+
+/**
+ * Build an asset accept transaction.
+ */
+public class AssetAcceptTransactionBuilder<T extends AssetAcceptTransactionBuilder<T>> extends TransactionBuilder<T> {
+    protected Address acceptingAccount = null;
+    protected BigInteger assetIndex = null;
+
+    /**
+     * Initialize a {@link AssetAcceptTransactionBuilder}.
+     */
+    public static AssetAcceptTransactionBuilder<?> Builder() {
+        return new AssetAcceptTransactionBuilder<>();
+    }
+
+    protected AssetAcceptTransactionBuilder() {
```

Comment:
Comment.
     * Creates a tx to mark the account as willing to accept the asset.
Required fields: 
...
Optional fields:
...